apps/src/dom.js

```diff
@@ -7,15 +7,15 @@ exports.addReadyListener = function(callback) {
 };
 
 exports.getText = function(node) {
-  return node.innerText || node.textContent;
+  return node.textContent || node.innerText;
 };
 
 exports.setText = function(node, string) {
-  if (node.innerText) {
-    node.innerText = string;
-  } else {
-    node.textContent = string;
-  }
+  // innerText is not supported in Firefox
+  // textContent is not supported in IE8
+  // See: http://caniuse.com/#feat=textcontent
+  //      http://caniuse.com/#feat=innertext
+  node.innerText = node.textContent = string;
```

Comment:
I'm not sure where dom.js is used, but since this function was already using both I figured we should continue to do so for best compatibility with IE8.  Other changes are in netsim- or applab-specific code, where we explicitly don't support IE8.

I suppose it couldn't hurt to use both in those places too, to give shared apps the best chance at IE8 compatibility.